pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala

```diff
@@ -63,14 +69,27 @@ case class MetsData(
           Left(new Exception(s"Couldn't match $accessCondition to a license"))
       }
     }.sequence
-  }
 
-  private def sourceIdentifier = {
+  private val parseAccessStatus: Either[Exception, Option[AccessStatus]] =
+    accessConditionStatus
+      .map(_.toLowerCase)
```

Comment:
Do we know that we have different versions of the casing in the data? This implies that.

Reply:
Nope have no idea what form the data takes tbh, I just did this as the data is typically pretty messy. Happy to change if you have a strong aversion to it (that recent bug with pattern matching and lower casing is still fresh in my mind, so that could be a reason to not have this).